bucket.go

```diff
@@ -343,7 +343,7 @@ func (bucket *Bucket) inTransaction(fn func(txn *sql.Tx) error) error {
 			if sqliteErrCode(err) == sqlite3.ErrBusy || sqliteErrCode(err) == sqlite3.ErrLocked {
 				continue // retry
 			} else {
-				logError("Transaction aborted with error %T %v", err, err)
+				//logError("Transaction aborted with error %T %v:\n%s", err, err, readableStackTrace(2))
```

Comment:
is this intended to be commented out?

Reply:
It was too noisy in practice (happens a lot in normal use with a `MissingError`) so I commented it out for the time being intending to downgrade it or maybe just ignore certain errors.

I'll re-enable it but at `debug` level.